app/components/collectionPaperNote/collectionPaperNote.scss

```diff
@@ -0,0 +1,86 @@
+%common-note {
```

Comment:
Could you explain what this `%` selector means?

Reply:
% works ### as a `variable` in `sass/scss` syntax.